net-vpn/i2pd/i2pd-2.32.1.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd cmake toolchain-funcs
+
+DESCRIPTION="A C++ daemon for accessing the I2P anonymous network"
+HOMEPAGE="https://github.com/PurpleI2P/i2pd"
+SRC_URI="https://github.com/PurpleI2P/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="cpu_flags_x86_aes cpu_flags_x86_avx i2p-hardening libressl static +upnp"
+
+RDEPEND="
+	acct-user/i2pd
+	acct-group/i2pd
+	!static? (
+		dev-libs/boost:=[threads]
+		!libressl? ( dev-libs/openssl:0=[-bindist] )
+		libressl? ( dev-libs/libressl:0= )
+		upnp? ( net-libs/miniupnpc )
+	)"
+DEPEND="${RDEPEND}
+	static? (
+		dev-libs/boost:=[static-libs,threads]
+		!libressl? ( dev-libs/openssl:0=[static-libs] )
+		libressl? ( dev-libs/libressl:0=[static-libs] )
+		sys-libs/zlib:=[static-libs]
```

Comment:
Move this above libressl blocks, and remove `:=`.

Reply:
Hi. `CMAKE_USE_DIR="${S}/build"` works as it is now, after the `inherit cmake` statement. I tested the ebuilds before submitting this PR. It is hard to tell if it should be like this or not, at least from the manuals.
I can do the cosmetics, no problem. Why remove `:=` in zlib, though?

Reply:
> It is hard to tell if it should be like this or not, at least from the manuals.

I agree. Could you check whether this can be found on any Qt/Plasma ebuilds, and see how they do it? They're pretty much the cmake folks.

> I can do the cosmetics, no problem. Why remove `:=` in zlib, though?

Because zlib doesn't have a changing soname / broken ABI. The subslot addition for it was a mistake.

Reply:
I looked at a few random ebuilds where `CMAKE_USE_DIR` is used. They all define `CMAKE_PROFILE_GENERATOR` before `inherit`, but `CMAKE_USE_DIR` after. But I did not find any such ebuilds related to Qt/Plasma.
OK, I will remove `:=`